src/containers/User/UserArea.tsx

```diff
@@ -176,6 +194,40 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({ templates, dataViews }) => {
             />
           </List.Item>
         )}
+        {extReferenceDocs.length && (
```

Comment:
not related to changes in this PR but can you please
* add `selectOnBlur={false}` to all of the `<Dropdown` (apart from the reference docs), it's quite annoying that something is selected when menu looses focus

Reply:
Yeah, good call. ec1e5d4d